src/transaction/makeOutput.js

```diff
@@ -10,10 +10,20 @@ export default function makeOutput(condition, amount = '1') {
     if (typeof amount !== 'string') {
         throw new TypeError('`amount` must be of type string')
     }
+    const publicKeys = []
+    const getPublicKeys = details => {
+        if (details.type === 'ed25519-sha-256') {
+            if (publicKeys.indexOf(details.public_key) === -1) {
```

Comment:
There is this seksiness now: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes